trollflow2/launcher.py

```diff
@@ -438,3 +440,61 @@ def sendmail(config, trace):
     pid = Popen([sendmail, "-t", "-oi"], stdin=PIPE)
     pid.communicate(msg.as_bytes())
     pid.terminate()
+
+
+def launch(args_in):
```

Comment:
This function should be tested.

Reply:
Fixed in https://github.com/pytroll/trollflow2/pull/159/commits/7bfb65180c8d5dda8cd6449757d9966edb3cbabc